src/main/resources/templates/fragments/inputs/radio.html

```diff
@@ -19,5 +20,9 @@
             aria-invalid=${hasError},
             data-follow-up=${followUpId}">
     <span th:text="${label}"></span>
+    <p th:if="${hasHelpText}"
+       th:id="${inputName} + '-' + ${value} + '-help-text'"
+       th:utext="${radioHelpText}"
+       class="text--help" style="margin-bottom: 0px"></p>
```

Comment:
I think this could be `class="text--help with-no-padding"`

Reply:
Nice! Thanks.